swupd/manifest.go

```diff
@@ -467,6 +468,77 @@ func (m *Manifest) newDeleted(df *File) {
 	m.Files = append(m.Files, df)
 }
 
+// TODO: Consolidate this function with linkPeersAndChange.
+
+// TODO: Double check minVersion handling in linkPeersAndChange: if old=20, minversion=30 and
+// new=40, we are always setting to 40, but what if 30 exists?
```

Comment:
Double checked. Is not an issue because when setting versions (calling linkPeersAndChange, we are calling with the directly previous Manifest, so if we can't use that version, we use the new manifest one).

I'll remove this TODO.